src/styles/main.scss

```diff
@@ -1,3 +1,83 @@
+@import "./variables.scss";
+
 body {
   margin: 0;
 }
+
+* {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
```

Comment:
Just set the font for the whole page and inherit where needed. No need to use * selector here